DEVELOPMENT.md

```diff
@@ -9,21 +9,47 @@ The Pixie project currently consists of three main components:
 This document outlines the process for setting up the development environment for each of these components.
 
 ## Setting up the Environment
+There are two different ways to run a development environment. The first and easiest
```

Comment:
Can you add `--network host` to the run_docker.sh file ? This way, minikube should work and we won't have to make any particular recommendations of which Kubernetes platform uses which development environment here. 

docker run --rm -it \
  "${configs[@]}" \
  -v /var/run/docker.sock:/var/run/docker.sock \
  -v "${workspace_root}:/pl/src/px.dev/pixie" \
  --network host \
  "${PX_RUN_DOCKER_EXTRA_ARGS[@]}" \
  "${docker_image_with_tag}" \
  "${exec_cmd[@]}"